cogs/info.py

```diff
@@ -15,7 +15,52 @@ def __init__(self, bot):
         self.config = default.config()
         self.process = psutil.Process(os.getpid())
 
-    ## Proof of concept for use with @story{9}
+    ## Poll command @story{63}
+    # The poll command provides users with a command in which they can
+    # submit a question and two answer options which are then presented
+    # to the channel in the form of a question, two answers and an emoji
+    # button for each answer.
+    #
+    # A poll is really just a Message object that contains a formated 
+    # string (the message) with a populated list of Reaction objects. 
+    # For the purposes of this class, a poll is a Message object, which
+    # contains a formatted string (the message) and two reactions.
+    @commands.command()
```

Comment:
Will this get called when the user types "!poll"? Maybe I had been doing something wrong, but I think you may need an alias within the @commands.commad() in order for this to actually get called (unless this is getting called somewhere that I am not seeing). So something like, @commands.command(aliases=['poll'])

Reply:
`@commands.command()` by default uses the name of the function itself as the command name. Many pre-existing commands are designed this way. Aliases are if you want multiple names for the same command (like `!eightball` and `!8ball`)

Reply:
Oh, I was unaware of how that really worked... Thank you for clearing that up.